app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+
+    parts = command.split(" ")
```

Comment:
`" "` is default value for `split`
```suggestion
    parts = command.split()
```